pontoon/machinery/views.py

```diff
@@ -134,6 +134,53 @@ def microsoft_translator(request):
         return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
 
 
+def google_translate(request):
+    """Get translation from Google machine translation service."""
+    try:
+        text = request.GET['text']
+        locale_code = request.GET['locale']
+    except MultiValueDictKeyError as e:
+        return HttpResponseBadRequest('Bad Request: {error}'.format(error=e))
```

Comment:
Possible improvement: move this code into a `Form` and share it between `google` and `microsoft` views.

Reply:
See: https://github.com/mozilla/pontoon/pull/1104#discussion_r226265539.